meta-mender-imx/templates/local.conf.append

```diff
@@ -0,0 +1,18 @@
+
+# Appended fragment from meta-mender-community/meta-mender-imx/templates
+IMAGE_CLASSES += "mender-setup-imx"
+
+# General Settings
+MENDER_BOOT_PART_SIZE_MB_imx8mnevk = "256"
+MENDER_STORAGE_TOTAL_SIZE_MB_imx8mnevk = "1024"
+
+# Boot from SD
+#MENDER_STORAGE_DEVICE_imx8mnevk = "/dev/mmcblk1"
+
+# Boot from eMMC
+MENDER_STORAGE_DEVICE_imx8mnevk = "/dev/mmcblk2"
+
+IMAGE_FSTYPES_remove = "tar.bz2 ext4 sdcard.bz2 wic.bmap wic.bz2 uefiimg.bz2"
+IMAGE_BOOT_FILES_remove_imx8mnevk = " ${KERNEL_IMAGETYPE} ${@make_dtb_boot_files(d)} "
+MENDER_IMAGE_BOOTLOADER_FILE_imx8mnevk = "imx-boot"
+
```

Comment:
These could probably be in `mender-setup-imx.bbclass` but not critical.

Reply:
Done